### PR TITLE
JAMES-2586 Fix PostgresAuthenticationTest

### PR DESCRIPTION
**Pull request by hungphan227**

### CODE REVIEW

**Review by Arsnael — APPROVED** [review]

File: server/container/guice/common/src/test/java/org/apache/james/JamesServerExtension.java [hunk 1]
```diff
@@ -56,11 +56,22 @@ interface AwaitCondition {
 
     public enum Lifecycle {
         // Restarts the server for each class, including nested classes
-        PER_CLASS(JamesServerExtension::start, (extension, context) -> { }, (extension, context) -> { }, JamesServerExtension::stop),
+        PER_CLASS(
+            (extension, context) -> {
+                extension.setup(context);
+                extension.start(context);
```
Comment (quantranhong1999) [inline server/container/guice/common/src/test/java/org/apache/james/JamesServerExtension.java:62]: Can we open this change for the master branch also to see the tests result?

For postgresql branch, only Postgres tests are being executed.

**Comment by Arsnael** [comment]
Please rebase

File: server/container/guice/common/src/test/java/org/apache/james/JamesServerExtension.java (hunk 1 above)
Comment (Arsnael) [reply server/container/guice/common/src/test/java/org/apache/james/JamesServerExtension.java:62]: @hungphan227 can you take care of this please?

Comment (hungphan227) [reply server/container/guice/common/src/test/java/org/apache/james/JamesServerExtension.java:62]: https://github.com/apache/james-project/pull/2086

**Pull request merged**